packer/install-dependencies.sh

```diff
@@ -48,12 +48,8 @@ cat << EOF >> ${RELEASE_NOTES_FILEPATH}
 EOF
 
 if [[ ${FEATURE_FLAGS} == *"docker-engine"* ]]; then
-    DOCKER_ENGINE_REPO="https://apt.dockerproject.org/repo"
-    installDockerEngine
-    overrideDockerEngineStorageDriver
-    echo "  - docker-engine v${DOCKER_ENGINE_VERSION}" >> ${RELEASE_NOTES_FILEPATH}
-    installGPUDrivers
```

Comment:
Why are we not pre-pulling GPU drivers in VHD anymore?

Reply:
I'm rethinking this, we'll download (as appropriate) prereq's during VHD CI so we don't have to during VHD bootstrapping,.